docs/en/concepts-and-designs/service-hierarchy.md

```diff
@@ -11,12 +11,17 @@ There 2 ways to detect the connections:
 
 ### Automatically Matching 
 
-| Upper layer | Lower layer  | Matching rule                                                         |
-|-------------|--------------|-----------------------------------------------------------------------|
-| MESH        | MESH_DP      | upper service name equals lower service name                          |
-| MESH        | K8S_SERVICE  | upper service short name equals lower service short name              |
-| MESH_DP     | K8S_SERVICE  | upper service short name equals lower service short name              |
-| GENERAL     | K8S_SERVICE  | upper service short name equals lower service name without namespace  |
+| Upper layer       | Lower layer  | Matching rule                                                                         |
+|-------------------|--------------|---------------------------------------------------------------------------------------|
+| MESH              | MESH_DP      | upper service name equals lower service name                                          |
+| MESH              | K8S_SERVICE  | upper service short name equals lower service short name                              |
+| MESH_DP           | K8S_SERVICE  | upper service short name equals lower service short name                              |
+| GENERAL           | K8S_SERVICE  | upper service short name equals lower service name without namespace                  |
+| MYSQL             | K8S_SERVICE  | upper service name without namespace equals lower service short name                  |
+| POSTGRESQL        | K8S_SERVICE  | upper service name without namespace equals lower service name                        |
+| SO11Y_OAP         | K8S_SERVICE  | upper service name without namespace equals lower service name                        |
+| VIRTUAL_DATABASE  | MYSQL        | upper service short name remove port is equal to lower service short name  with fqdn  |
+| VIRTUAL_DATABASE  | POSTGRESQL   | upper service short name remove port is equal to lower service short name  with fqdn  |
```

Comment:
```suggestion
| VIRTUAL_DATABASE  | POSTGRESQL   | upper service short name remove port is equal to lower service short name  with fqdn  |

### Misidentifications & False Positives
...

#### Misidentifications between Virtual Database layer and MySQL/PostgreSQL layer
```

@wankai123 As we have known the limitation of hierarchy relationship between VIRTUAL_DATABASE and MYSQL/POSTGRESQL, we should provide a description here